### PR TITLE
Fixed inplace frontend testing and torch frontend inplace handling

### PR DESCRIPTION
**Pull request by AnnaTz**

### CODE REVIEW

**Review by vedpatwardhan — APPROVED** [review]
As far as the framework goes, we're good to merge. Thanks @AnnaTz 🙂

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/helpers/function_testing.py [hunk 1]
```diff
@@ -697,12 +698,13 @@ def test_frontend_function(
         else:
             # the function provides inplace update by default
             # check if returned reference is inputted reference
+            copy_kwargs["as_ivy_arrays"] = False
             first_array = ivy.func_wrapper._get_first_array(
                 *args, array_fn=array_fn, **kwargs
             )
             ret_ = get_frontend_ret(frontend_fn, *args, **kwargs)
             assert first_array is ret_
-            args, kwargs = copy_args, copy_kwargs
```
Comment (CatB1t) [inline ivy_tests/test_ivy/helpers/function_testing.py:705]: I think this line existed so it `args` and `kwargs` gets reset to what they were before arguments were modified in place, is that not the case anymore?

**Review by CatB1t — CHANGES_REQUESTED** [review]
Hello @AnnaTz, lgtm! just a small comment then we're good to go!

File: ivy_tests/test_ivy/helpers/function_testing.py (hunk 1 above)
Comment (AnnaTz) [reply ivy_tests/test_ivy/helpers/function_testing.py:705]: They appear to be unused 
image

Comment (CatB1t) [reply ivy_tests/test_ivy/helpers/function_testing.py:705]: In that case, feel free to remove it, good work @AnnaTz !

**Review by CatB1t — APPROVED** [review]

**Pull request merged**